git-hub

```diff
@@ -2204,7 +2203,7 @@ if __name__ == '__main__':
                 for e in err['errors']:
                     if 'message' in err:
                         errf('{}{message}',
-                            ' ' * len(prefix), **e)
+                                ' ' * len(prefix), **e)
```

Comment:
I wonder if this line can join the line above (concatenate them) as it might look better

Reply:
Agreed, https://github.com/sociomantic-tsunami/git-hub/pull/272/commits/3c099c443067b74da474efa09c8e9ecd54a9d2f1